components/schema-migrator/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-The Schema Migrator is responsible for database schema migrations.
+The Schema Migrator is responsible for provisioner's database schema migrations.
```

Comment:
```suggestion
The Schema Migrator is responsible for Runtime Provisioner's database schema migrations.
```
I guess that's the provisioner we're referring to? For consistency, I'll use the same term as in line 21